stingray/events.py

```diff
@@ -0,0 +1,212 @@
+"""
+Definition of :class:`EventList`.
+
+:class:`EventList` is used to handle photon arrival times.
+"""
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from stingray.simulator.events import events_to_lc, lc_to_events, assign_energies
+
```

Comment:
Maybe these three functions could go into helper methods in the class itself? Or do you need them elsewhere? It would make it easier for anyone trying to look up what they do without having to search for another script.